File: Beginner_Projects/Patterns/Alphabats/Python/M.py
for row in range(5):
    for col in range(7):
        if col == 0 or col == 6 or (row+col == 6 and row != 4) or (row-col == 0 and row != 4):
            print("*", end=" ")
        else:
            print(end="  ")
    print()
input()

File: Beginner_Projects/Patterns/Alphabats/C++/M.cpp
#include <iostream>
#include <conio.h>
using namespace std;

int main()
{
    for (int row = 0; row < 5; row++)
    {
        for (int col = 0; col < 7; col++)
        {
            if (col == 0 || col == 6 || ((col + row == 6) && row != 4) || ((row - col == 0) && row != 4))
            {
                cout << "*";
            }
            else
            {
                cout << " ";
            }
        }
        cout << endl;
    }
    getch();
}

// Created by Salfi Hacker Mansoor Bukhari